web/src/app/multiplayer/page.tsx

```diff
@@ -25,11 +25,10 @@ export default function Multiplayer() {
 
     const handleSearchChange = (event: React.ChangeEvent<HTMLInputElement>) => {
         
-        //TODO: Maybe implement a better search function?? Kinda trash/wonky
-        const { value } = event.currentTarget;
+        const value = event.target.value;
         setSearch(value);
```

Comment:
I think we can delete this search state variable now right?

Reply:
I think so, will doublecheck.

Reply:
We don't need this but this also means we remove the search state variable on the TextInput which does look weird, but we don't need it so I guess we remove it.

Reply:
doesn't look weird, the default value for a textinput is just what the user typed in

Reply:
No I'm talking about on the textinput, it doesn't have a input={} value